airflow/ti_deps/deps/base_ti_dep.py

```diff
@@ -47,7 +47,7 @@ def __hash__(self):
         return hash(type(self))
 
     def __repr__(self):
-        return "<TIDep({self.name})>".format(self=self)
+        return f"<TIDep({self.name})>"
```

Comment:
For consistency, let's use `self.__class__.__name__` here too

Reply:
This one is a bit tricky. `self.name` is actually returning the `NAME` field that is defined in most of the `TIDep` class with fallback to `self.__class__.__name__`. 

I was actually thinking to remove that `NAME` field because it seems redundant. That being said, I haven't taken a deeper look into how is that `NAME` field being used by other modules. So It will be great if you or anyone can provide some background information on that. Or I will double check that tomorrow.

https://github.com/apache/airflow/blob/d8482c32695e173c026d22633d48ec93f69e58e8/airflow/ti_deps/deps/base_ti_dep.py#L52-L58